backend/modules/professor/routers/professor_grading.py

```
@@ -41,7 +41,7 @@ async def get_submission_details(
         if not instance.cohort_assignment:
             raise HTTPException(status_code=403, detail="Access denied")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)